File: OPENAPI-REST-API/openapi-client/php-dt/src/App/DTO/ServerOrderBandwidth.php
<?php
declare(strict_types=1);

namespace App\DTO;

use Articus\DataTransfer\PhpAttribute as DTA;

/**
 * A bandwidth option available when ordering a dedicated server.
 */
class ServerOrderBandwidth
{
    /**
     * Bandwidth ID.
     */
    #[DTA\Data(field: "id", nullable: true)]
    #[DTA\Validator("Scalar", ["type" => "string"])]
    public string|null $id = null;

    /**
     * Bandwidth price.
     */
    #[DTA\Data(field: "price", nullable: true)]
    #[DTA\Validator("Scalar", ["type" => "int"])]
    public int|null $price = null;

    /**
     * Bandwidth image.
     */
    #[DTA\Data(field: "img", nullable: true)]
    #[DTA\Validator("Scalar", ["type" => "string"])]
    public string|null $img = null;

    /**
     * Short description of the bandwidth.
     */
    #[DTA\Data(field: "short_desc", nullable: true)]
    #[DTA\Validator("Scalar", ["type" => "string"])]
    public string|null $short_desc = null;

    /**
     * Long description of the bandwidth.
     */
    #[DTA\Data(field: "long_desc", nullable: true)]
    #[DTA\Validator("Scalar", ["type" => "string"])]
    public string|null $long_desc = null;

    /**
     * Bandwidth type.
     */
    #[DTA\Data(field: "type", nullable: true)]
    #[DTA\Validator("Scalar", ["type" => "string"])]
    public string|null $type = null;

    /**
     * Quantity of bandwidth.
     */
    #[DTA\Data(field: "qty", nullable: true)]
    #[DTA\Validator("Scalar", ["type" => "string"])]
    public string|null $qty = null;

    /**
     * Active status.
     */
    #[DTA\Data(field: "active", nullable: true)]
    #[DTA\Validator("Scalar", ["type" => "string"])]
    public string|null $active = null;

    /**
     * Monthly price.
     */
    #[DTA\Data(field: "monthly_price", nullable: true)]
    #[DTA\Validator("Scalar", ["type" => "int"])]
    public int|null $monthly_price = null;

    /**
     * Display of bandwidth price.
     */
    #[DTA\Data(field: "price_display", nullable: true)]
    #[DTA\Validator("Scalar", ["type" => "string"])]
    public string|null $price_display = null;

    /**
     * Display of monthly bandwidth price.
     */
    #[DTA\Data(field: "monthly_price_display", nullable: true)]
    #[DTA\Validator("Scalar", ["type" => "string"])]
    public string|null $monthly_price_display = null;

}

File: OPENAPI-REST-API/swagger-client/scala-akka-http-server/src/main/scala/io/swagger/server/model/ServerOrderBandwidth.scala
package io.swagger.server.model


/**
 * A bandwidth option available when ordering a dedicated server.
 *
 * @param id Bandwidth ID. for example: ''15''
 * @param price Bandwidth price. for example: ''0''
 * @param img Bandwidth image. for example: ''bandwidth.jpg''
 * @param short_desc Short description of the bandwidth. for example: ''150TB (10Gb Port)''
 * @param long_desc Long description of the bandwidth. for example: '' - 2000GB Bandwidth...''
 * @param `type` Bandwidth type. for example: ''Standard''
 * @param qty Quantity of bandwidth. for example: ''4000''
 * @param active Active status. for example: ''1''
 * @param monthly_price Monthly price. for example: ''15''
 * @param price_display Display of bandwidth price. for example: ''$0.00''
 * @param monthly_price_display Display of monthly bandwidth price. for example: ''$15.00''
 */
case class ServerOrderBandwidth (
  id: Option[String],
  price: Option[Int],
  img: Option[String],
  short_desc: Option[String],
  long_desc: Option[String],
  `type`: Option[String],
  qty: Option[String],
  active: Option[String],
  monthly_price: Option[Int],
  price_display: Option[String],
  monthly_price_display: Option[String]
)

